dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/QdrantDataStore.hold

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Memory;
+using Microsoft.SemanticKernel.Memory.Storage;
+using Qdrant.DotNet;
+using Qdrant.DotNet.Internal.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Skills.Memory.VectorDB;
+
+/// <summary>
+/// /// An implementation of <see cref="IDataStore{TValue}"/> backed by a Qdrant Vector database.
+/// </summary>
+/// <remarks>The vector data is saved to Qdrant Vector Database run locally or in cloud, specified in the constructor.
+/// The data persists between subsequent instances. </remarks>
+/// <typeparam name="TValue">The type of data to be stored in this data store.</typeparam>
+public class QdrantDataStore<TValue> : IDataStore<TValue>
```

Comment:
Qdrant will only ever hold vectors. There should be no QdrantDataStore, only QDrantMemoryStore

Reply:
This was a merge issue this file is being removed. Thanks for the catch.